src/main/java/com/google/cloud/storage/Database.java

```diff
@@ -0,0 +1,46 @@
+package com.google.cloud.storage;
+
+import java.util.HashMap;
+
+
+/* This class creates an instance of a database that a file can be written to and read from.
+Assumptions: for weach runID, only 1 of each datatype can be associated with it */
+public class Database{
```

Comment:
I think you may need to update the code by our yesterday's discussion.

Reply:
will be meeting today with David to discuss changes!